### PR TITLE
Added Tests for get_page in api

### PR DESCRIPTION
**Pull request by DartingMelody**
## General Checkups
- [x ] Have you checked that there aren't other open pull requests for the same issue/update/change?
- [x ] If your code includes new features and not just bug fixes/copy editing: Did you include new tests?

### CODE REVIEW

**Comment by DartingMelody** [comment]
hi @madprime , Please review the PR. Thanks

File: ohapi/cassettes/test_get_page_with_results.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept: ['*/*']
+      Accept-Encoding: ['gzip, deflate']
+      Connection: [keep-alive]
+      User-Agent: [python-requests/2.9.1]
+    method: GET
+    uri: https://www.openhumans.org/api/direct-sharing/project/exchange-member/?access_token=dQ5Phvwaz2kthlKnv0I4ytulMt8fBG
```
Comment (madprime) [inline ohapi/cassettes/test_get_page_with_results.yaml]: the access_token should be filtered out and replaced with a placeholder

File: ohapi/tests/test_get_page.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import vcr
+import unittest
+import ohapi.api as api
+
+my_vcr = vcr.VCR(path_transformer=vcr.VCR.ensure_suffix('.yaml'),
+                 cassette_library_dir='ohapi/cassettes',
+                 filter_query_parameters=['access_token'])
+
+
+class test_get_page(unittest.TestCase):
```
Comment (madprime) [inline ohapi/tests/test_get_page.py]: I'd prefer for `test_api.py` to have all tests relevant to `api.py`. Could you please move this class to that file?

Also, I'd like the class to be renamed to conform to PEP8 (https://www.python.org/dev/peps/pep-0008/#class-names), e.g. `APITestGetPage`

**Comment by DartingMelody** [comment]
Hi @gedankenstuecke @madprime I have renamed and moved the tests for get_page function. I have also  set filter query parameters as suggested. However, there is one hound violation which is occurring because of a locals() function call. Please review.

**Comment by madprime** [comment]
@DartingMelody Thanks, looks good! No worries on Hound's issue due to use of locals(), that's expected. :)

**Pull request merged**